cpp/include/cudf/strings/substring.hpp

```diff
@@ -197,6 +197,10 @@ std::unique_ptr<column> slice_strings(
   size_type count,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+std::unique_ptr<cudf::column> json_to_array(
+  cudf::strings_column_view const& col,
+  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
```

Comment:
I'd rather this go in to a `json.hpp` header than `substring.hpp` header.

Reply:
I ever wonder that whether the json stuff should go to the IO module instead of string module (@jrhemstad ?).